packages/containers-windows/containers-windows.2.8.1/opam

```diff
@@ -7,7 +7,7 @@ build: [
 depends: [
   "dune"
   "dune-windows"
-  "dune-configurator"
+  "dune-configurator-windows"
```

Comment:
I don't think that this'll work actually. The configurator is always executed in the host's context.